test/admission_test.go

```diff
@@ -29,11 +31,14 @@ func TestAdmissionControl(t *testing.T) {
 				backendType: backendType,
 				endpointParameters: []string{
 					"admission-control-policy=limit",
-					"admission-control-policy-limit-max-concurrent-txn=600",
+					fmt.Sprintf("admission-control-policy-limit-max-concurrent-txn=%d", writeLimit),
```

Comment:
Any particular scenario causing you to drop this limit?

Reply:
This might actually be a good candidate for a separate PR.

The problem is that this test creates hundreds of goroutines just to make sure that admission control blocks some queries. However the same check can be achieved with a smaller number without loosing correctness.

This change brings the execution of the test from minutes (on my machine) to less then a second. And in general, it is important to be able to run tests frequently while developing.